backend/tracim_backend/lib/core/user.py

```diff
@@ -79,6 +89,10 @@
 from tracim_backend.models.userconfig import UserConfig
 
 KNOWN_MEMBERS_ITEMS_LIMIT = 5
+AVATAR_RATIO = ImageRatio(1, 1)
+COVER_RATIO = ImageRatio(35, 4)
+DEFAULT_AVATAR_SIZE = ImageSize(100, 100)
+DEFAULT_COVER_SIZE = ImageSize(1300, 150)
```

Comment:
Ratio and default size written as constants. I don't think it's necessary to make this configurable now.
100x100 is the biggest avatar size we do support in frontend (we do support also 20x20, 40x40, 50x50).

For cover ration, 35/4 mean 8,75. This is ok for big cover but this lead to not completely accurate result for lower size image (for a 100x10 image, this mean a result of 87x10 which is ok but not exactly the same ratio). It's a edge case, so not a big deal but using a more simple ratio will lead to slighly better result for low sized image/preview (as preview doesn't change the ratio).